app/_src/policies/meshtrafficpermission_experimental.md

```diff
@@ -0,0 +1,141 @@
+---
+title: MeshTrafficPermission
+---
+
+{% tip %}
+[Mutual TLS](/docs/{{ page.release }}/policies/mutual-tls) has to be enabled to make MeshTrafficPermission work.
+{% endtip %}
+
+## Overview
+
+`MeshTrafficPermission` policy defines which clients are allowed or denied access to services inside a mesh based on their identities.
+By default, if no policy is present, all requests are denied.
+
+It enables:
+
+* denying all requests by default, or blocking specific clients (by SPIFFE ID)
+  so that no service owner can override them
+* allowing groups of clients, such as everything in the `observability` namespace,
+  to access all services by default, while still letting individual services opt out
+* giving service owners control to allow specific clients, and the ability to block abusive ones
+  even if they were previously allowed
+
+Here is a common example:
+
+{% policy_yaml %}
+```yaml
+type: MeshTrafficPermission
+name: my-app-permissions
+mesh: my-mesh
+spec:
+  targetRef:
+    kind: Dataplane
+    labels:
+      app: my-app
+  rules:
+    - default:
+        deny:
+          - spiffeId:
+              type: Prefix
+              value: "spiffe://other-mesh.us-east-2.mesh.local/ns/legacy-ns"
```

Comment:
is the cross mesh common example? I don't think so, maybe we should stick with single mesh